otelcollector/build/windows/scripts/main.ps1

```diff
@@ -367,8 +367,15 @@ function Set-CertificateForME {
     Copy-Item -r /etc/config/settings/akv /opt/akv-copy
 
     Get-ChildItem "C:\etc\config\settings\akv\" |  Foreach-Object {
-        if (!($_.Name.startswith('..'))) {
-            Import-PfxCertificate -FilePath $_.FullName -CertStoreLocation Cert:\CurrentUser\My > $null
+        # check if child is a file and not a directory
```

Comment:
kaveesh - what was the issue here ? we are trying to import anything in there ? But everything in there should be certs, isnt it ?

Reply:
The below list is what we're iterating on to import. Previously it only had the PFX file and the directories so I was ignoring the directories and importing the certificate. Now it looks like two null .crt and .key files are also being mounted in the container which throw an error when I try to import them since they're null. I've update it so that I try to import any non null file within the said folder.

![image](https://user-images.githubusercontent.com/28612268/216149084-ee12104f-df82-472d-90e8-557ce9e269a3.png)

Reply:
ok. thanks. is this only an issue on windows ?

Reply:
thanks.